server/lib/transferwise.ts

```diff
@@ -30,11 +38,13 @@ const axios = Axios.create({
 const compactRecipientDetails = <T>(object: T): Partial<T> => omitBy(object, isNull);
 const getData = <T extends { data?: object }>(obj: T | undefined): T['data'] | undefined => obj && obj.data;
 
-const getAxiosError = (error: AxiosError): string => {
-  if (error.response) {
+const getErrorCode = (error: AxiosError): string => {
+  if (error.response?.data?.errorCode) {
+    return error.response.data.errorCode;
+  } else if (error.response?.status) {
     // The request was made and the server responded with a status code
     // that falls out of the range of 2xx
-    return `${error.response.status}: ${JSON.stringify(error.response.data)}`;
+    return `${error.response.status}: ${JSON.stringify(error.response.data.errorCode)}`;
```

Comment:
**[:no_entry_sign: **Blocker**]** If `error.response.data.errorCode` is set, it will always enter the first `if` and be `undefined` here no?

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
Nice catch, it was supposed to serialize data.